packages/marketplace/src/sagas/admin-stats.ts

```diff
@@ -28,15 +27,21 @@ export const adminStatsDataFetch = function*({ data }) {
         .toDate()
         .toISOString()
     }
-    const response = yield call(fetcher, {
-      url: `${url}?${setQueryParams({
-        pageSize: GET_ALL_PAGE_SIZE,
-        ...queryParams,
-      })}`,
-      api: window.reapit.config.marketplaceApiUrl,
-      method: 'GET',
-      headers: generateHeader(window.reapit.config.marketplaceApiKey),
-    })
+    const serviceToCall =
+      area === 'APPS'
```

Comment:
I think should move to switch case or use hash map to get the condition instead of if else too much
https://medium.com/front-end-weekly/switch-case-if-else-or-a-lookup-map-a-study-case-de1c801d944

Reply:
```
const services = {
DEVELOPERS: fetchDevelopersList,
INSTALLATIONS: fetchInstallationsList,
APPS: fetchAppsList,
}
const response = yield call(serviceToCall[area], { pageSize: GET_ALL_PAGE_SIZE, ...queryParams })
```